dev/src/write-batch.ts

```diff
@@ -124,6 +124,8 @@ interface WriteOp {
   precondition?: api.IPrecondition | null;
 }
 
+export class UpdateBuilder {}
```

Comment:
I don't see any usages of `UpdateBuilder`.  If this is unused, can it be deleted?